### DISCUSSION TITLE
How to extract the font properties of specific text?

### DISCUSSION THREAD

**Discussion by meghanaviyyapu** [pymupdf/PyMuPDF#1512] [Looking for help] [answered]
Can you let me know how to extract the font size and font name of only some part of text  in the PDF?

**Comment by JorjMcKie** [comment, marked as answer]
If you do `page.get_text("dict")["blocks"]`, then each text block (one with `block["type"] == 0`) is a dictionary containing a list of line dictionaries, with in turn a list of sspan dictionaries.
This hierarchy of dictionaries can be looked up [here](https://pymupdf.readthedocs.io/en/latest/textpage.html#structure-of-dictionary-outputs).
The span dictionaries contain the font name and size of the respective text portion - along with the rectangle containing that text.

So you can either select spans falling inside your area, or you can let PyMuPDF select only that part of the output intersecting your area: `page.get_text("dict", clip=area)...`.

**Reply by meghanaviyyapu** [reply]
Thank you.
